certs/postgres/init.go

```diff
@@ -6,56 +6,13 @@
 package postgres
 
 import (
-	"errors"
-	"fmt"
-
 	_ "github.com/jackc/pgx/v5/stdlib" // required for SQL access
```

Comment:
Is `_ "github.com/jackc/pgx/v5/stdlib" // required for SQL access` really needed? It's most likely already loaded in the db package.